helm/ph-ee-test/values.yaml

```diff
@@ -0,0 +1,507 @@
+ph-ee-engine:
+
+  camunda-platform:
+    global:
+      zeebe: "zeebetest-zeebe"
+    zeebe:
```

Comment:
you have to use the overridefullname property and use it. the word zeebetest is coming from the release name and you do not have any control over it when someone else is using it in a different cluster